tests/testthat/_snaps/expect-all.md

```diff
@@ -0,0 +1,56 @@
+# validates its inputs
+
+    Code
+      expect_all_equal(mean, 1)
+    Condition
+      Error in `expect_all_equal()`:
+      ! `object` must be a vector, not a function.
+    Code
+      expect_all_equal(logical(), 1)
+    Condition
+      Error in `expect_all_equal()`:
+      ! `object` must not be empty.
+    Code
+      expect_all_equal(1:10, mean)
+    Condition
+      Error in `expect_all_equal()`:
+      ! `expected` must be a vector, not a function.
+    Code
+      expect_all_equal(1:10, 1:2)
+    Condition
+      Error in `expect_all_equal()`:
+      ! `expected` must be length 1.
+
+# can compare atomic vectors
+
+    Code
+      expect_all_equal(x, TRUE)
+    Condition
+      Error:
+      ! Expected every element of `x` to equal TRUE.
```

Comment:
```suggestion
      ! Expected every element of `x` to equal `TRUE`.
```

??

Reply:
FWIW that would be inconsistent with `expect_equal()`.